core/src/components/textarea/textarea.ionic.scss

```diff
@@ -166,3 +174,41 @@ ion-icon {
 :host(.has-focus) .textarea-highlight {
   transform: scale(1);
 }
+
+// Textarea - Disabled
+// ----------------------------------------------------------------
+
+:host(.textarea-disabled) {
+  // color for the text within the textarea
+  --color: #{globals.$ion-primitives-neutral-500};
+  --background: #{globals.$ion-primitives-neutral-100};
+  --border-color: #{globals.$ion-primitives-neutral-300};
+  --placeholder-color: #{globals.$ion-primitives-neutral-500};
+
+  pointer-events: none;
+}
+
+:host(.textarea-disabled:not(.ion-valid)) .textarea-bottom .helper-text,
+:host(.textarea-disabled) .textarea-bottom .counter,
+:host(.textarea-disabled) .label-text-wrapper {
+  color: globals.$ion-primitives-neutral-500;
```

Comment:
The `ion-icon` is missing in this list. The designs also show that the icons would be neutral 500 when textarea is disabled.